include/OpenSpaceToolkit/Astrodynamics/Trajectory/Orbit/Model/Kepler/COE.hpp

```diff
@@ -454,6 +454,88 @@ class COE
     /// @return Local Time of the Descending Node (LTDN) in hours
     static Time ComputeLTDN(const Angle& raan, const Instant& anInstant, const Sun& sun = Sun::Default());
 
+    /// @brief Compute the Sun-synchronous inclination for a given semi-major axis and eccentricity
+    ///
+    /// @ref https://github.com/JuliaSpace/SatelliteAnalysis.jl/blob/main/src/sun_synchronous_orbits.jl
+    ///
+    /// @param aSemiMajorAxis A semi-major axis
+    /// @param anEccentricity An eccentricity
+    /// @param aCelestialObjectSPtr A shared pointer to a central celestial body
+    /// @return Sun-synchronous inclination
+    static Angle ComputeSunSynchronousInclination(
+        const Length& aSemiMajorAxis, const Real& anEccentricity, const Shared<const Celestial>& aCelestialObjectSPtr
+    );
+
+    /// @brief Compute the Right Ascension of the Ascending Node (RAAN) from Local Time of the Ascending Node (LTAN)
+    ///
+    /// @param aLocalTimeAtAscendingNode A local time at ascending node
+    /// @param anEpoch An epoch
+    /// @param sun A Sun model
+    /// @return Right Ascension of the Ascending Node
+    static Angle ComputeRaanFromLTAN(
+        const Time& aLocalTimeAtAscendingNode, const Instant& anEpoch, const Sun& sun = Sun::Default()
+    );
+
+    /// @brief Construct a Sun-synchronous COE
+    ///
+    /// @param aSemiMajorAxis A semi-major axis
+    /// @param aLocalTimeAtAscendingNode A local time at ascending node
+    /// @param anEpoch An epoch
+    /// @param aCelestialObjectSPtr A shared pointer to a central celestial body
+    /// @param anEccentricity An eccentricity
+    /// @param anArgumentOfLatitude An argument of latitude
+    /// @return COE
+    static COE SunSynchronous(
+        const Length& aSemiMajorAxis,
+        const Time& aLocalTimeAtAscendingNode,
+        const Instant& anEpoch,
+        const Shared<const Celestial>& aCelestialObjectSPtr,
+        const Real& anEccentricity = Real::Zero(),
+        const Angle& anArgumentOfLatitude = Angle::Zero()
+    );
+
+    /// @brief Construct a Geosynchronous COE
+    ///
+    /// @param anEpoch An epoch
+    /// @param anInclination An inclination
+    /// @param aLongitude A longitude above the surface
+    /// @param aCelestialObjectSPtr A shared pointer to a central celestial body
+    /// @return COE
+    static COE GeoSynchronous(
+        const Instant& anEpoch,
+        const Angle& anInclination,
+        const Angle& aLongitude,
+        const Shared<const Celestial>& aCelestialObjectSPtr
+    );
```

Comment:
Would it make sense to default inclination like it's been done for other params in the other constructors?

```suggestion
    static COE GeoSynchronous(
        const Instant& anEpoch,
        const Angle& aLongitude,
        const Shared<const Celestial>& aCelestialObjectSPtr,
        const Angle& anInclination = Angle::Zero()
    );
```